__tests__/pathToDotNotation.test.js

```diff
@@ -0,0 +1,14 @@
+const pathToDotNotation = require('../build.js').pathToDotNotation
+
+describe('Token formatters: PascalCase', () => {
+  it('returns a valid token name in PascalCase', () => {
```

Comment:
oof, my bad.. my suggestion earlier was for the wrong file...

this is the correction for this file..

```suggestion
describe('Token formatters: dot notation', () => {
  it('returns a valid token name in dot notation', () => {
```

sorry about that @lukasoppermann 😅

Reply:
I think I fixed it already when I added the blocks.